skipruntime-ts/adapters/postgres/src/index.ts

```diff
@@ -230,17 +230,19 @@ FOR EACH ROW EXECUTE FUNCTION %I();`,
   }
 
   unsubscribe(instance: string): void {
-    this.client
-      .query(format("DROP FUNCTION IF EXISTS %I CASCADE;", instance))
-      .then(
-        () => this.open_instances.delete(instance),
-        (e: unknown) => {
-          console.error(
-            "Error unsubscribing from resource instance: " + instance,
-          );
-          throw e;
-        },
-      );
+    if (this.open_instances.has(instance))
+      this.client
+        .query(format("DROP FUNCTION IF EXISTS %I CASCADE;", instance))
+        .then(
+          () => this.open_instances.delete(instance),
+          (e: unknown) => {
+            console.error(
+              "Error unsubscribing from resource instance: " + instance,
+            );
+            console.log(e);
```

Comment:
Do we want to log before throwing here? If so, do we want to do it consistently throughout the postgres adapter?

Reply:
Nope, that's debugging code that I should strip back out...